flux-errors/locales/en-US/refineck.ftl

```diff
@@ -28,6 +28,9 @@ refineck_fold_error =
 refineck_unknown_error =
     unknown error
```

Comment:
while we are at it, maybe we should change this to something less intimidating. Something like "the safety of this statement cannot be proven.

Reply:
Btw, I like `Tag::Other` for prototyping, but eventually, everything should be moved out of it. But no reason to completely throw off users while we transition.